core/src/trezor/ui/confirm.py

```diff
@@ -256,7 +256,7 @@ def _on_loader_start(self) -> None:
         # Loader has either started growing, or returned to the 0-position.
         # In the first case we need to clear the content leftovers, in the latter
         # we need to render the content again.
-        ui.display.bar(0, 0, ui.WIDTH, ui.HEIGHT - 60, ui.BG)
+        ui.display.bar(0, 0, ui.WIDTH, ui.HEIGHT - 58, ui.BG)
```

Comment:
Is this intentional? Why do we need that?

Reply:
Yes, this seems to be a previously uncatched error. If the last row of the HoldToConfirm dialog contains letter "g" - i.e. some pixels are rendered below the text baseline, they stay on screen when user unpushes the button before it is confirmed. I think this was an oversight before and 58 is better value.